tests/Behat/DrupalContext.php

```diff
@@ -201,4 +202,64 @@ protected function getLanguageIdByName(string $name): string {
     throw new \InvalidArgumentException("Language '{$name}' not found.");
   }
 
+  /**
+   * Redirect user on Node creation page.
+   *
+   * @param string $content_type_name
+   *   Content type name.
+   *
+   * @Given I am visiting the :content_type_name create node page
+   */
+  public function iAmVisitingTheCreateNodePage(string $content_type_name): void {
+    $node_bundle = $this->getEntityTypeByLabel($content_type_name);
+    $this->visitPath('node/add/' . $node_bundle);
+  }
+
+  /**
+   * Check that selectbox with node language selector is hidden.
```

Comment:
The function should allow to find any selectbox, so the comment should reflect that (and not point out to the language selector)